byron/ledger/executable-spec/src/Ledger/Update/Generators.hs

```diff
@@ -1,15 +1,16 @@
+{-# LANGUAGE NamedFieldPuns      #-}
```

Comment:
Looks like this change is not being used?

Reply:
Indeed! I'll remove it!